frontend/pages/beta/create-new-model.tsx

```diff
@@ -0,0 +1,97 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Card from '@mui/material/Card'
+import Divider from '@mui/material/Divider'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Radio from '@mui/material/Radio'
+import RadioGroup from '@mui/material/RadioGroup'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import { useState } from 'react'
+
+import Wrapper from '../../src/Wrapper'
```

Comment:
Should we use absolute imports for this, so that we can keep our templates the same no matter what level the page is at?  An import like...

```
import Wrapper from '@src/Wrapper'
```

Might be better.

Reply:
Isn't working unfortunately, think we need to update the config somewhere but I agree this change would be nice